### ISSUE TITLE
kafka-python 1.3.3 breaks aiokafka producer?

### ISSUE THREAD

**Issue by ZmeiGorynych** [aio-libs/aiokafka#163]
Hi,

when I run the following code using kafka-python 1.3.1, it runs fine. 
```
from kafka.common import KafkaError
from aiokafka import AIOKafkaConsumer, AIOKafkaProducer
import asyncio
topic = 'test_topic'
def produce(loop):
    # Just adds message to sending queue
    future = yield from producer.send(topic, b'some_message_bytes')
    resp = yield from future
    print("Message produced: partition {}; offset {}".format(
        resp.partition, resp.offset))

loop = asyncio.get_event_loop()
producer = AIOKafkaProducer(loop=loop, bootstrap_servers='192.168.99.100:9092')
# Bootstrap client, will get initial cluster metadata
loop.run_until_complete(producer.start())
loop.run_until_complete(produce(loop))
loop.run_until_complete(producer.stop())
```
Yet when I update to kafka-python 1.3.3, it produces the error below. 
Could you please take a look?
Thanks a lot!
```
AttributeError                            Traceback (most recent call last)
<ipython-input-1-81a83ca85581> in <module>()
     14 # Bootstrap client, will get initial cluster metadata
     15 loop.run_until_complete(producer.start())
---> 16 loop.run_until_complete(produce(loop))
     17 loop.run_until_complete(producer.stop())

C:\Users\Egor\Anaconda2\envs\py3k\lib\asyncio\base_events.py in run_until_complete(self, future)
    385             raise RuntimeError('Event loop stopped before Future completed.')
    386 
--> 387         return future.result()
    388 
    389     def stop(self):

C:\Users\Egor\Anaconda2\envs\py3k\lib\asyncio\futures.py in result(self)
    272             self._tb_logger = None
    273         if self._exception is not None:
--> 274             raise self._exception
    275         return self._result
    276 

C:\Users\Egor\Anaconda2\envs\py3k\lib\asyncio\tasks.py in _step(***failed resolving arguments***)
    237                 # We use the `send` method directly, because coroutines
    238                 # don't have `__iter__` and `__next__` methods.
--> 239                 result = coro.send(None)
    240             else:
    241                 result = coro.throw(exc)

<ipython-input-1-81a83ca85581> in produce(loop)
      5 def produce(loop):
      6     # Just adds message to sending queue
----> 7     future = yield from producer.send(topic, b'some_message_bytes')
      8     resp = yield from future
      9     print("Message produced: partition {}; offset {}".format(

C:\Users\Egor\Anaconda2\envs\py3k\lib\site-packages\aiokafka\producer.py in send(self, topic, value, key, partition)
    277 
    278         fut = yield from self._message_accumulator.add_message(
--> 279             tp, key_bytes, value_bytes, self._request_timeout_ms / 1000)
    280         return fut
    281 

C:\Users\Egor\Anaconda2\envs\py3k\lib\site-packages\aiokafka\message_accumulator.py in add_message(self, tp, key, value, timeout)
    197                 self._wait_data_future.set_result(None)
    198 
--> 199         future = batch.append(key, value)
    200         if future is None:
    201             # Batch is full, can't append data atm,

C:\Users\Egor\Anaconda2\envs\py3k\lib\site-packages\aiokafka\message_accumulator.py in append(self, key, value)
     83             return None
     84 
---> 85         encoded = Message(value, key=key, magic=self._version_id).encode()
     86         msg = Int64.encode(self._relative_offset) + Int32.encode(len(encoded))
     87         msg += encoded

C:\Users\Egor\Anaconda2\envs\py3k\lib\site-packages\kafka\util.py in __call__(self, *args, **kwargs)
    151         Calls the method on target with args and kwargs.
    152         """
--> 153         return self.method()(self.target(), *args, **kwargs)
    154 
    155     def __hash__(self):

C:\Users\Egor\Anaconda2\envs\py3k\lib\site-packages\kafka\protocol\message.py in _encode_self(self, recalc_crc)
     71 
     72     def _encode_self(self, recalc_crc=True):
---> 73         version = self.magic
     74         if version == 1:
     75             fields = (self.crc, self.magic, self.attributes, self.timestamp, self.key, self.value)

AttributeError: 'NoneType' object has no attribute 'magic'
```

**Comment by Drizzt1991** [comment]
Yes, that is correct, 1.2 version is only compatible with 1.3.1. The master branch was fixed to be compatible with 1.3.3 and will be released as 1.3 ~ at the end of the month. If you really need it I can do a patch for 1.2 that supports 1.3.3.

**Comment by ZmeiGorynych** [comment]
Thanks for the quick reply! 
It's fine, I can stick to kafka-python 1.3.1 until you release the fix.

**Comment by tvoinarovskyi** [comment]
Will port the fix into 0.2 branch and plan to release tomorrow. Sorry for the wait.

**Comment by tvoinarovskyi** [comment]
Fixed in 0.2.3, released
